### DISCUSSION TITLE
connecting a yamaha crw2200 to a V4.1 adaptor

### DISCUSSION THREAD

**Discussion by JGCK8** [keirf/greaseweazle#423] [Q&A]
how to connect a 5.25 " floppy drive with 40 pins to a V4.1 adapter with 34 pins/

**Comment by keirf** [comment]
5.25" floppy drives don't have 40 pins. It's probably an optical drive.

**Comment by JGCK8** [comment]
The floppy drive is YAMAHA CRW2200.

Sent from Mail for Windows

**Reply by keirf** [reply]
Yes, that's a CD-RW drive. Greaseweazle works only with magnetic floppy drives. So there is no way to connect your CRW2200 drive, end of story I'm afraid.

**Comment by JGCK8** [comment]
Hi, Thanks for the info.  Would you help me in canceling  the shipment and initiating
A refund?
Thanks,  John

Sent from Mail for Windows

**Reply by keirf** [reply]
Contact the company you purchased from. If that's me, it will be via my eBay UK store.
